packages/edit-site/src/components/sidebar-navigation-screen-template/home-template-details.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { debounce } from '@wordpress/compose';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import {
+	CheckboxControl,
+	__experimentalUseNavigator as useNavigator,
+	Button,
+	Icon,
+	__experimentalInputControl as InputControl,
+	__experimentalTruncate as Truncate,
+} from '@wordpress/components';
+import { header, footer, layout, chevronRightSmall } from '@wordpress/icons';
+import { useMemo, useState, useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SidebarNavigationScreenDetailsPanel,
+	SidebarNavigationScreenDetailsPanelRow,
+} from '../sidebar-navigation-screen-details-panel';
+import { unlock } from '../../lock-unlock';
+import { store as editSiteStore } from '../../store';
+import { useLink } from '../routes/link';
+
+const EMPTY_OBJECT = {};
+
+function TemplateAreaButton( { area, postId, icon, title } ) {
+	const icons = {
+		header,
+		footer,
+	};
+	// Generic area labels - not the same as the template part title defined in "templateParts" in theme.json.
+	const areaLabels = {
+		header: __( 'Header' ),
+		footer: __( 'Footer' ),
+	};
```

Comment:
Are there only ever these two area labels, or is this extensible by themes at all?

Reply:
The design looks like this:

<img width="300" src="https://user-images.githubusercontent.com/846565/244014781-8f225c9d-eb83-47c8-8dd2-e2ce9b669b79.png" />

I'm guessing it's a generic label for the template part, at least the "official" ones that can be in a template. 

So in that sense, it's incomplete.

But we already have the `title` (displayed on the right), which comes from theme.json (if it's set), so it seems redundant (?)

Reply:
The label on the left was intended to be the semantic area which can be useful when the template part name doesn't provide that information. That said we can probably make do with the icon and omit the label if it helps:

<img width="372" alt="Screenshot 2023-06-13 at 11 23 32" src="https://github.com/WordPress/gutenberg/assets/846565/3c40e18d-06b7-4781-b174-abede5289e95">

Reply:
Thanks for the updated design! Done.